src/table/table.scss

```diff
@@ -155,6 +156,12 @@
   }
 }
 
+@mixin iui-table-header-actions-container {
+  display: flex;
+  flex-grow: 1;
```

Comment:
Why this one is needed?

Reply:
Without it does not take full width
![image](https://user-images.githubusercontent.com/81580355/167694139-7d958855-b7cf-4ab8-b9cb-6e6530a47b98.png)